.github/workflows/publish-docker-image-for-eth.yml

```
@@ -1,28 +1,28 @@
-    steps:
-      - uses: actions/checkout@v4
-      - name: Setup repo
-        uses: ./.github/actions/setup-repo
-        id: setup
-        with:
-          docker-username: ${{ secrets.DOCKER_USERNAME }}
-          docker-password: ${{ secrets.DOCKER_PASSWORD }}
-          docker-remote-multi-platform: true
-          docker-arm-host: ${{ secrets.ARM_RUNNER_HOSTNAME }}
-          docker-arm-host-key: ${{ secrets.ARM_RUNNER_KEY }}
+    steps: wethERC20 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)